testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java

```diff
@@ -29,4 +33,11 @@ public String sendEvent(@RequestParam(required = true) String subject) {
         return String.format("Sent MOTECH event: %s with empty params: %s", subject, eventParams.toString());
     }
 
+    @RequestMapping(value = "/sendSMS202NIP47", method = RequestMethod.POST, headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @ResponseBody
+    public String sendSMS202NIP47(@RequestBody String template) {
+        return "OK";
```

Comment:
@srivastavaAbhi There seems to be multiple versions of this mock responder api for 200/202 in place in multiple PRs. Can you help consolidate?